<?xml version="1.0" encoding="UTF-8"?>


<ui version="4.0">


 <class>MainWindow</class>


 <widget class="QMainWindow" name="MainWindow">


  <property name="geometry">


   <rect>


    <x>0</x>


    <y>0</y>


    <width>898</width>


    <height>600</height>


   </rect>


  </property>


  <property name="windowTitle">


   <string>Steganograaphy and Encryption System</string>


  </property>


  <widget class="QWidget" name="centralwidget">


   <widget class="QLabel" name="label">


    <property name="geometry">


     <rect>


      <x>20</x>


      <y>10</y>


      <width>121</width>


      <height>16</height>


     </rect>


    </property>


    <property name="text">


     <string>Input/Ouput String</string>


    </property>


   </widget>


   <widget class="QTextEdit" name="TextIO">


    <property name="geometry">


     <rect>


      <x>40</x>


      <y>30</y>


      <width>801</width>


      <height>81</height>


     </rect>


    </property>


   </widget>


   <widget class="QLabel" name="label_2">


    <property name="geometry">


     <rect>


      <x>30</x>


      <y>140</y>


      <width>71</width>


      <height>16</height>


     </rect>


    </property>


    <property name="text">


     <string>Password</string>


    </property>


   </widget>


   <widget class="QLineEdit" name="Password_Text">


    <property name="geometry">


     <rect>


      <x>120</x>


      <y>140</y>


      <width>461</width>


      <height>21</height>


     </rect>


    </property>


   </widget>


   <widget class="QLabel" name="label_3">


    <property name="geometry">


     <rect>


      <x>30</x>


      <y>180</y>


      <width>71</width>


      <height>16</height>


     </rect>


    </property>


    <property name="text">


     <string>Containers</string>


    </property>


   </widget>


   <widget class="QLineEdit" name="Container_Text">


    <property name="geometry">


     <rect>


      <x>120</x>


      <y>180</y>


      <width>461</width>


      <height>21</height>


     </rect>


    </property>


    <property name="clearButtonEnabled">


     <bool>true</bool>


    </property>


   </widget>


   <widget class="QLineEdit" name="Secret_Text">


    <property name="geometry">


     <rect>


      <x>120</x>


      <y>220</y>


      <width>461</width>


      <height>21</height>


     </rect>


    </property>


    <property name="clearButtonEnabled">


     <bool>true</bool>


    </property>


   </widget>


   <widget class="QLabel" name="label_4">


    <property name="geometry">


     <rect>


      <x>30</x>


      <y>220</y>


      <width>61</width>


      <height>16</height>


     </rect>


    </property>


    <property name="text">


     <string>Secret File</string>


    </property>


   </widget>


   <widget class="QLabel" name="label_5">


    <property name="geometry">


     <rect>


      <x>30</x>


      <y>260</y>


      <width>53</width>


      <height>15</height>


     </rect>


    </property>


    <property name="text">


     <string>Capacity</string>


    </property>


   </widget>


   <widget class="QLabel" name="CapaInfo">


    <property name="geometry">


     <rect>


      <x>120</x>


      <y>260</y>


      <width>461</width>


      <height>16</height>


     </rect>


    </property>


    <property name="text">


     <string>Capcatiy Info</string>


    </property>


   </widget>


   <widget class="QPushButton" name="Check_Button">


    <property name="geometry">


     <rect>


      <x>600</x>


      <y>250</y>


      <width>131</width>


      <height>23</height>


     </rect>


    </property>


    <property name="text">


     <string>Check Capcacity</string>


    </property>


   </widget>


   <widget class="QGroupBox" name="groupBox">


    <property name="geometry">


     <rect>


      <x>50</x>


      <y>290</y>


      <width>120</width>


      <height>80</height>


     </rect>


    </property>


    <property name="title">


     <string>En/Decode</string>


    </property>


    <widget class="QRadioButton" name="radioButton">


     <property name="geometry">


      <rect>


       <x>10</x>


       <y>20</y>


       <width>95</width>


       <height>19</height>


      </rect>


     </property>


     <property name="text">


      <string>Encode</string>


     </property>


    </widget>


    <widget class="QRadioButton" name="radioButton_2">


     <property name="geometry">


      <rect>


       <x>10</x>


       <y>50</y>


       <width>95</width>


       <height>19</height>


      </rect>


     </property>


     <property name="text">


      <string>Decode</string>


     </property>


    </widget>


   </widget>


   <widget class="QGroupBox" name="groupBox_2">


    <property name="geometry">


     <rect>


      <x>240</x>


      <y>290</y>


      <width>120</width>


      <height>80</height>


     </rect>


    </property>


    <property name="title">


     <string>Type</string>


    </property>


    <widget class="QRadioButton" name="radioButton_3">


     <property name="geometry">


      <rect>


       <x>10</x>


       <y>20</y>


       <width>95</width>


       <height>19</height>


      </rect>


     </property>


     <property name="text">


      <string>File</string>


     </property>


    </widget>


    <widget class="QRadioButton" name="radioButton_4">


     <property name="geometry">


      <rect>


       <x>10</x>


       <y>50</y>


       <width>95</width>


       <height>19</height>


      </rect>


     </property>


     <property name="text">


      <string>String</string>


     </property>


    </widget>


   </widget>


   <widget class="QPushButton" name="pushButton_2">


    <property name="geometry">


     <rect>


      <x>290</x>


      <y>390</y>


      <width>91</width>


      <height>51</height>


     </rect>


    </property>


    <property name="text">


     <string>Start</string>


    </property>


   </widget>


   <widget class="QPushButton" name="pushButton_3">


    <property name="geometry">


     <rect>


      <x>430</x>


      <y>390</y>


      <width>91</width>


      <height>51</height>


     </rect>


    </property>


    <property name="text">


     <string>Clear</string>


    </property>


   </widget>


   <widget class="QPushButton" name="SelectContainer">


    <property name="geometry">


     <rect>


      <x>600</x>


      <y>180</y>


      <width>75</width>


      <height>23</height>


     </rect>


    </property>


    <property name="text">


     <string>Select</string>


    </property>


   </widget>


   <widget class="QPushButton" name="SelectSecret">


    <property name="geometry">


     <rect>


      <x>600</x>


      <y>220</y>


      <width>75</width>


      <height>23</height>


     </rect>


    </property>


    <property name="text">


     <string>Select</string>


    </property>


   </widget>


   <widget class="QPushButton" name="adv_btn">


    <property name="geometry">


     <rect>


      <x>600</x>


      <y>320</y>


      <width>131</width>


      <height>41</height>


     </rect>


    </property>


    <property name="text">


     <string>Advanced Settings</string>


    </property>


   </widget>


  </widget>


  <widget class="QMenuBar" name="menubar">


   <property name="geometry">


    <rect>


     <x>0</x>


     <y>0</y>


     <width>898</width>


     <height>21</height>


    </rect>


   </property>


   <widget class="QMenu" name="menuFile">


    <property name="title">


     <string>File</string>


    </property>


    <addaction name="actionSelect_Container"/>


    <addaction name="actionSelect_Secret"/>


    <addaction name="separator"/>


    <addaction name="actionExit"/>


   </widget>


   <widget class="QMenu" name="menuOptions">


    <property name="title">


     <string>Options</string>


    </property>


    <addaction name="actionAdvanced_Settings"/>


   </widget>


   <widget class="QMenu" name="menuTools">


    <property name="title">


     <string>Tools</string>


    </property>


   </widget>


   <widget class="QMenu" name="menuHelp">


    <property name="title">


     <string>Help</string>


    </property>


    <addaction name="actionAbout"/>


   </widget>


   <addaction name="menuFile"/>


   <addaction name="menuOptions"/>


   <addaction name="menuTools"/>


   <addaction name="menuHelp"/>


  </widget>


  <widget class="QStatusBar" name="statusbar"/>


  <action name="actionExit">


   <property name="text">


    <string>Exit</string>


   </property>


  </action>


  <action name="actionAbout">


   <property name="text">


    <string>About</string>


   </property>


  </action>


  <action name="actionAdvanced_Settings">


   <property name="text">


    <string>Advanced Settings</string>


   </property>


  </action>


  <action name="actionSelect_Container">


   <property name="text">


    <string>Select Container</string>


   </property>


  </action>


  <action name="actionSelect_Secret">


   <property name="text">


    <string>Select Secret</string>


   </property>


  </action>


 </widget>


 <resources/>


 <connections/>


</ui>


